docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss

```diff
@@ -73,4 +73,8 @@
       padding: hb-calculate-rems(20px);
     }
   }
+
+  hr {
+    border: 0;
```

Comment:
@mndonx We're almost there, just a little extra thing to fix. The top and bottom margin is different in the `hr`. There are not explicit rules in the theme for that, but the browser is adding some default styles. I checked in multiple browsers and it's consistent, so it should be fixed adding this to the `hr` styles:

```
margin-block-start: 0.83em;
margin-block-end: 0.83em;
font-size: 3.6rem;
```

Reply:
@cienvaras Good eye, thank you! I also found another instance where the logic needed to be applied (in the original pattern.) Let me know if you spot anything else. Thank you!